algebra/primefield/src/prime_field.rs

```diff
@@ -9,7 +9,7 @@ use std::{
 };
 use zkp_u256::{
     AddInline, Binary, DivRem, Inv, Montgomery as _, MontgomeryParameters, MulInline, NegInline,
-    One, Pow, SquareInline, SubInline, Zero,
+    One, Pow, SquareInline, SubInline, Zero, U256,
```

Comment:
Would be better if we can keep the PrimeField implementation generic over the UInt type (which it currently is).